migrations/20200719172434-change-patients-barcodeValue-to-string.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
as it stands, this migration will wipe any existing values in the `barcodeValue` column - we don't want this to happen, so ideally you migrate those int values to string so we can keep the values still